pkgs/development/tools/language-servers/nixd/default.nix

```diff
@@ -0,0 +1,84 @@
+{ stdenv
```

Comment:
The usual ordering is 
1. lib
2. stdenv
3. fetchers
4. all the rest in ASCIIbetical order